programmers/pull-two-out-and-add.js

```diff
@@ -0,0 +1,15 @@
+function solution(numbers){
+    const numbers_len = numbers.length;
+    let sums = [];
+    for(let i = 0 ; i < numbers_len-1 ; i++){
+        for(let j = i+1 ; j<numbers_len ; j++){
+            sums.push(numbers[i]+numbers[j]);
+        }
+    }
+    sums = Array.from(new Set(sums));
+    return sums.sort(function (a,b){
+        return a-b
+    });
```

Comment:
ES6에 포함된 애로우펑션으로도 풀 수 있습니다.
애로우 펑션이 더 깔끔해보이지 않나요?
```suggestion
    return sums.sort((a,b) => {
        return a-b
    });
```